.gitlab-ci.yml

```diff
@@ -1,3 +1,5 @@
+image: golang:1.13
+
```

Comment:
@tranchitella For future reference: this actually will not work as expected.

Here you define the default image, but then the templates are overriding the image on each job. In this case there is not undesired effect as the image used by the templates is actually golang:1.13.

The correct way to do it is as it was before: first include the templates, then override the specific jobs that require a different version (or do nothing if we just want to use the "latest Mender supported version"). See documentation [here](https://github.com/mendersoftware/mendertesting/blob/master/.gitlab-ci-check-golang-unittests.yml).

Reply:
@lluiscampos thanks for clarifying it, do you want me to launch a PR to remove the useless image override?

Reply:
A bit late here, but I assume you got that as I have reworked the templates usage I have cleaned up these. So nothing to do, it was only a heads-up to avoid confusion in the future.